package.json

```diff
@@ -28,10 +28,10 @@
   },
   "homepage": "https://github.com/leaverou/eleventy-plugin-citations#readme",
   "devDependencies": {
-    "@11ty/eleventy": "^3.0.0-alpha.14",
     "release-it": "^17.4.1"
   },
   "dependencies": {
+    "@11ty/eleventy": "^3.0.0-alpha.14",
```

Comment:
This was actually intentional. If I didn't include 11ty at all in dependencies, I was not able to import things from it. But by adding 11ty to devDependencies, npm does not install whatever version of 11ty I have specified here when the plugin is used, but instead uses the host code’s version of 11ty. And since this is an 11ty plugin, there will _always_ be a version of 11ty. 😁

Reply:
Ah! I thought it was just because you never imported from it at all, but that makes total sense. Change reverted 😄